screenapiexample/src/androidTest/java/net/gini/android/vision/screen/testhelper/PermissionsHelper.java

```diff
@@ -0,0 +1,85 @@
+package net.gini.android.vision.screen.testhelper;
+
+import static android.support.test.InstrumentationRegistry.getInstrumentation;
+import static android.support.test.InstrumentationRegistry.getTargetContext;
+
+import android.Manifest;
+import android.annotation.TargetApi;
+import android.app.UiAutomation;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.os.ParcelFileDescriptor;
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class PermissionsHelper {
```

Comment:
FYI: The Android Testing Support Library 1.0 comes with a [GrantPermissionRule](https://developer.android.com/reference/android/support/test/rule/GrantPermissionRule.html)